README.md

```diff
@@ -109,7 +109,21 @@ ks param set kubeflow-core usageId $(uuidgen)
 ks apply default -c kubeflow-core
 ```
 
-The above command sets up JupyterHub and a custom resource for running TensorFlow training jobs. Furthermore, the ksonnet packages
+In order to access the Jupyter notebook, there are several considerations based on the target Kubernetes cluster. In the simple case where the deployment is tested on a single Kubernetes node (e.g. on an on-premise machine) without access to a loadbalancer, then these two lines is used to access the Jupyter notebook: 
```

Comment:
I'm not quite sure where this information should go. I don't think it belongs here in the README.md
because this is intended to be a quick_start and not go into to many edge cases.

user_guide.md is probably a better fit.  But even then its not really clear to me where it should go since there's really two bits of information corresponding to two different sections

1. Deploying Kubeflow 
2. Connecting to Jupyter

How about starting a section in user_guide.md "On Prem deployments" and then put this information there.